draft-thoji-scone-trone-protocol.md

```diff
@@ -186,9 +186,16 @@ signaling occurs as this is specific to the application in use.
 A signal does not prove that a higher rate would not be successful.  Endpoints
 that receive this signal therefore need to treat the information as advisory.
 
-As an advisory signal, network elements cannot assume that endpoints will
-respect the signal.  Though this might reduce the need for more active rate
-limiting, how rate limit enforcement is applied is a matter for network policy.
+The fact that an endpoint requests bitrate signals does not necessarily mean
+that it will adhere to them; in some cases, the endpoint cannot. In certain
+situations, a flow may initially be used to serve video chunks, with the client
+selecting appropriate chunks based on bitrate signals, but later switch to a
+bulk download for which bitrate adaptation is not applicable. In other cases,
+the flow may be a tunnel carrying multiple connections, some of which may not be
+capable of handling SCONE signals. Therefore, when a network element detects a
+flow using more bandwidth than advertised via SCONE, it SHOULD fall back to rate
+limiting based on congestion control signals, as if the flow were not using
+SCONE at all.
```

Comment:
```suggestion
flow using more bandwidth than advertised via SCONE, congestion control
signals might be necessary to ensure that the flow does not exceed the
limits that are set in policy.
```

Reply:
Thank you for the review. Other changes made total sense, but I'm not fully certain about this one.

While it is true that CC might be necessary to force flows to be in policy, the proposed change does not address the original issue; i.e., that network elements should not punish non-adhering flows more so than non-SCONE flows. If network elements start behaving as such, endpoint developers would have no choice but to disable SCONE.

What about something like: "congestion control signals can be used to ensure that the flow performs on par with non-SCONE flows."?

Reply:
@martinthomson Went ahead with 387b028, please let me know what you think.